src/dataplane.ts

```diff
@@ -1,70 +1,88 @@
-import { satisfies } from 'semver'
-
 import { humanReadableDate } from '@/helpers'
 import { ONLINE, OFFLINE, PARTIALLY_DEGRADED, KUMA_ZONE_TAG_NAME } from '@/consts'
 import Kuma from '@/services/kuma'
-import { Dataplane, LabelValue, DataplaneInsight, DataplaneOverview, DiscoverySubscription } from '@/types'
-
-/*
-dpTags takes a Dataplane received from backend and construct the list of tags in form of array of objects with label and value.
-It flattens common tags so we don't display them twice.
-
-Takes a Dataplane received from backend and construct the list of tags in form of array of objects with label and value.
-It flattens common tags so we don't display them twice.
-@example
-
-type: Dataplane
-mesh: default
-name: cluster-1.ingress-01
-networking:
-  inbound:
-    - port: 1234
-      tags:
-        kuma.io/service: backend
-        version: 1
-    - port: 1235
-      tags:
-        kuma.io/service: backend-api
-        version: 1
-
-Will produce:
-[
-  { label: 'kuma.io/service', value: 'backend'},
-  { label: 'kuma.io/service', value: 'backend-api'},
-  { label: 'version', value: '1'},
-]
- */
+import {
+  Compatibility,
+  DataPlane,
+  RawDataPlane,
+  LabelValue,
+  DataplaneInsight,
+  DataplaneOverview,
+  DiscoverySubscription,
+  DataPlaneEntityMtls,
+} from '@/types'
 
 type TODO = any
 
-export function dpTags(dataplane: Dataplane): LabelValue[] {
-  let tags: TODO[] = []
+/**
+ * List of data plane tag labels that are considered important enough to be shown in the data plane list view.
+ */
+export const IMPORTANT_DATA_PLANE_TAG_LABELS: string[] = [
+  'kuma.io/protocol',
+  'kuma.io/service',
+  'kuma.io/zone',
+]
+
+/**
+ * Takes a data plane and constructs the list of tags. It flattens common tags so we don't display them twice.
```

Comment:
This seems to contradict your example

Reply:
The way this utility was implemented considers uniqueness of tags as label-value pairs meaning that a network inbound tag `kuma.io/service: frontend` and a network outbound tag `kuma.io/service: backend` are distinct.

Reply:
Ok might be worth clarifying the comment.

Reply:
I’ve added a note to it.